StartLogDrain.php

```
@@ -0,0 +1,221 @@
+            $readme_path = $config_path.'/README.md';
+            $command = [
+                "echo 'Saving configuration'",
+                "mkdir -p $config_path",
+                "echo '{$parsers}' | base64 -d | tee $parsers_config > /dev/null",
+                "echo '{$config}' | base64 -d | tee $fluent_bit_config > /dev/null",
+                "echo '{$compose}' | base64 -d | tee $compose_path > /dev/null",
+                "echo '{$readme}' | base64 -d | tee $readme_path > /dev/null",
+                "test -f $config_path/.env && rm $config_path/.env",
+            ];
+            if ($type === 'newrelic') {
+                $add_envs_command = [
```

PHPMD (2.15.0)

The variable $add_envs_command is not named in camelCase. (undefined)

(CamelCaseVariableName)